e2e/test/scenarios/filters-reproductions/filters-reproductions.cy.spec.js

```diff
@@ -1199,3 +1199,40 @@ describe("issue 40622", () => {
     });
   }
 });
+
+describe("issue 44435", () => {
+  const longString =
+    "In aut numquam labore fuga. Et tempora sit et mollitia aut ullam et repellat. Aliquam sint tenetur culpa eius tenetur. Molestias ipsa est ut quisquam hic necessitatibus. Molestias maiores vero nesciunt.";
```

Comment:
The issue has not been fixed - try a long string without whitespaces.

Reply:
Also see screenshot linked [here](https://github.com/metabase/metabase/issues/44435#issuecomment-2208997560)

Reply:
Thank you so much.
Fixed the repro and skipped it.